01/highest-rated.md

```diff
@@ -0,0 +1,676 @@
+| `[]`                                                                         | `None`                                         |
+
+Select every statement that is true.
+
+##### !end-question
+##### !options
+
+* The function's name is `get_highest_rated`
+* Restaurant data is passed in to the function through parameters
+* Restaurant data is passed in to the function through the return value
+* The function returns a list of all restaurant data
+* The function returns the data of the highest-rated restaurant
+* The highest-rated restaurant is the dictionary with the biggest value of `name`
+* The highest-rated restaurant is the dictionary with the biggest value of `rating`
+* The highest-rated restaurant is the dictionary with the biggest value of `ratings`
+* If there are no restaurants, then the function returns `None`
+* If there are two or more highest-rated restaurants, then the function returns a list containing both restaurants
+* If there are two or more highest-rated restaurants, then the function returns `None`
+* If there are two or more highest-rated restaurants, the original problem statement doesn't mention what to return.
+
+##### !end-options
+##### !answer
+
+* The function's name is `get_highest_rated`
+* Restaurant data is passed in to the function through parameters
+* The function returns the data of the highest-rated restaurant
+* The highest-rated restaurant is the dictionary with the biggest value of `rating`
+* If there are no restaurants, then the function returns `None`
+* If there are two or more highest-rated restaurants, the original problem statement doesn't mention what to return.
+
+##### !end-answer
+##### !explanation
+
+According to the original problem statement and given examples:
+
+* The function `get_highest_rated` has one parameter, `restaurants`, which is a list of restaurants.
+* The highest-rated restaurant is the restaurant with the largest value of **rating** specifically. This problem statement does _not_ need to compare restaurant names, or restaurants themselves!
+* One of the examples says that if `restaurants` is an empty list, then the function should return `None`.
+* The original problem statement doesn't mention what to return if there are two or more highest-rated restaurants. We would need to make assumptions or ask questions to know what behavior is required here.
+
+##### !end-explanation
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 7 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: checkbox
+* id: d7586297
+* title: PSE
+##### !question
+
+Select all statements that are true about the input parameter, `restaurants`.
+
+##### !end-question
+##### !options
+
+* `restaurants` is a list of restaurants.
+* `restaurants` is a dictionary of restaurants.
+* Each restaurant is a list.
+* Each restaurant is a dictionary.
+* Each dictionary contains two key-value pairs: `name` and `rating`.
+* Each restaurant has a `name` field and a `rating` field.
```

Comment:
Hm.. in our[ dictionary lesson](https://learn-2.galvanize.com/cohorts/2835/blocks/1151/content_files/01-intro-to-python/07-dictionaries.md) we don't mention the term "field" anywhere. Just key and value with their synonyms address and content.

I imagine students referencing the dictionary lesson for this PSE so it would be good to terminology alignment with the lesson. So we can either:

1) Add 'field' as a synonym for key to the dictionary lesson OR
2) Remove this last option

I'm partial to #2 as I think most instructors say "key, value", but either works :)

Reply:
What do you think about changing it to

```suggestion
* Each restaurant has a `name` and a `rating`.
```

?